config/jobs/kubernetes/sig-k8s-infra/trusted/sig-security-trusted.yaml

```diff
@@ -36,7 +36,7 @@ periodics:
         fi
         echo "Running snyk scan .."
         EXIT_CODE=0
-        RESULT_UNFILTERED=$(snyk test -d --json) || EXIT_CODE=$?
+        RESULT_UNFILTERED=$(snyk test -d --json-file-outout=${ARTIFACTS}/snyk-test-unfiltered.json --json) || EXIT_CODE=$?
```

Comment:
`outout` ... `output`?

Reply:
The existing `--json` puts the json output to the stdout.  The new `--json-file-output` additionally creates a file with the same json for troubleshooting after the run.